scraper.rb

```diff
@@ -19,7 +19,9 @@ def scrape(h)
 
 start = 'http://www.parlament.mt/membersofparliament?l=1'
 data = scrape(start => MembersPage).member_urls.map do |url|
-  scrape(url => MemberPage).to_h.merge(term: 12)
+  scrape(url => MemberPage).to_h
+                           .reject { |k, _v| k == :electoral_history }
```

Comment:
That's a somewhat nasty way to delete an element from a Hash, but we can live with it…